boot/freeldr/bootsect/ntfs.S

```diff
@@ -0,0 +1,804 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Bootsector
+ * FILE:            boot/freeldr/bootsect/fat32.S
+ * PURPOSE:
+ * PROGRAMMERS:     Sylvain Deverre
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include <asm.inc>
+#include <freeldr/include/arch/pc/x86common.h>
+
+#define BP_REL(x) [bp+x-offset start]
+
+.code16
+
+//ORG HEX(7c00)
+
+start:
+    jmp short main
+    nop
+OEMName:
+    .ASCII "NTFS    "   // NTFS signature
+BytesPerSector:
+    .word 512          
+SectsPerCluster:
+    .byte 8
+ReservedSectors:
+    .word 0             // Unused by NTFS
+NumberOfFats:
+    .byte 0             // Unused by NTFS
+MaxRootEntries:
+    .word 0             // Unused by NTFS
+TotalSectors:
+    .word 0             // Unused by NTFS
+MediaDescriptor:
+    .byte HEX(0f8)
+SectorsPerFat:
+    .word 0             // Unused by NTFS
+SectorsPerTrack:
+    .word 0             // Should contain disk geometry
+NumberOfHeads:
+    .word 0             // Should contain disk geometry
+HiddenSectors:
+    .long 0
+TotalSectorsBig:
+    .long 0
```

Comment:
Unused by NTFS ?

Reply:
Those values are used by NTFS, but can't be computed at compile time (since it depends of partition scheme). The labels are here only to get correct addresses to access those values in the code.

Reply:
My comment was only about "TotalSectorsBig"; according to documentation (e.g. Starman's MBR/VBR stuff, and another website) the TotalSectorsBig doesn't seem to be used by NTFS.

Reply:
Yes, TotalSectorsBig is set to 0 on NTFS partitions formatted by Windows (and I was misled by Github display of PRs :p)